mods/cnc/sequences/misc.yaml

```diff
@@ -436,3 +436,36 @@ smokland:
 		Length: 32
 		Tick: 120
 		ZOffset: 1023
+
+airstrike:
+	cursor: mouse2
+		Start: 88
+		Length: 8
```

Comment:
The animation here plays faster than the cursor.  Will need to adjust Tick so they match.